src/models/User/createUser/index.js

```diff
@@ -21,16 +25,32 @@ type CreateUserResponse = {
 }
 
 const createUser = async ({
+  firstName,
+  lastName,
   email,
-  name,
+  job,
   passwordPlain
 }: CreateUserArgs): Promise<CreateUserResponse> => {
   const errors: Array<ValidationError> = []
-  const trimmedName = name.trim()
-  const nameValidation = validateName(trimmedName)
+  const trimmedFirstName = firstName.trim()
+  const firstNameValidation = validateFirstName(trimmedFirstName)
 
-  if (!nameValidation.isValid) {
-    errors.push({ field: 'name', message: nameValidation.error })
+  if (!firstNameValidation.isValid) {
+    errors.push({ field: 'name', message: firstNameValidation.error })
+  }
+
+  const trimmedLastName = lastName.trim()
+  const lastNameValidation = validateLastName(trimmedLastName)
+
+  if (!lastNameValidation.isValid) {
+    errors.push({ field: 'name', message: lastNameValidation.error })
+  }
+
+  const trimmedJob = job.trim()
+  const jobValidation = validateJob(trimmedJob)
+
+  if (!jobValidation.isValid) {
+    errors.push({ field: 'job', message: jobValidation.error })
   }
```

Comment:
That method is starting to get messy with all these validations… 😄
I feel like if we used Objection’s validations, we’d still have to do weird stuff to send our own errors, so this is fine to me.